tools/lint/README.md

```diff
@@ -0,0 +1,100 @@
+We have a lint tool for catching common mistakes in test files. Now it can
+check html, htm, xhtml, xhtm, json and xml files. You can run it manually
+by starting the `lint.py` executable from the root of your local
+crosswalk-test-suite working directory like this:
+
+```
+./tools/lint/lint.py
+```
+
+The lint tool is also run automatically for every submitted pull request,
+and reviewers will not merge branches with tests that have lint errors, so
+you must either [fix all lint errors](#fixing-lint-errors), or you must
+[white-list test files] (#updating-the-whitelist) to suppress the errors.
+
+## Fixing lint errors
+
+You must fix any errors the lint tool reports, unless an error is for
+something essential to a certain test or that for some other exceptional
+reason shouldn't prevent the test from being merged. In those cases you can
+[white-list test files](#updating-the-whiteslist) to suppress the errors.
+Otherwise, use the details in this section to fix all errors reported.
+
+* **CR AT EOL**: Test-file line ends with CR (U+000D) character; **fix**:
+  reformat file so each line just has LF (U+000A) line ending (standard,
+  cross-platform "Unix" line endings instead of, e.g., DOS line endings).
+
+* **EARLY-TESTHARNESSREPORT**: Test file has an instance of
+  `<script src='/resources/testharnessreport.js'>` prior to
+  `<script src='/resources/testharness.js'>`; **fix**: flip the order.
+
+* **INDENT TABS**: Test-file line starts with one or more tab characters;
+  **fix**: use spaces to replace any tab characters at beginning of lines.
+
+* **MISSING-TESTHARNESSREPORT**: Test file is missing an instance of
+  `<script src='/resources/testharnessreport.js'>`; **fix**: ensure each
+  test file contains `<script src='/resources/testharnessreport.js'>`.
```

Comment:
Test file is missing an instance of  `<script src='/resources/testharnessreport.js'>` if there is testharness.js.
Also add **MISSING-TESTHARNESS**.